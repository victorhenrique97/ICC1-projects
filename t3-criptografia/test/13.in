2.bin
2
