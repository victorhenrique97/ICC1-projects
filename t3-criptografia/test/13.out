Se nao usar memoria Heap a gente te tira nota, em.
